package.json

```diff
@@ -80,7 +80,6 @@
     "drizzle-kit": "^0.27.2",
     "eslint": "^9.15.0",
     "eslint-config-prettier": "^9.1.0",
-    "eslint-config-ts": "^4.0.1",
```

Comment:
minor cleanup: this was throwing peer dependency warnings -- and wasn't actually being used anywhere.